docs/generated/packages/node/executors/node.json

```diff
@@ -17,48 +17,56 @@
       "buildTargetOptions": {
         "type": "object",
         "description": "Additional options to pass into the build target.",
-        "default": {}
+        "default": {},
+        "x-priority": "important"
       },
       "waitUntilTargets": {
         "type": "array",
         "description": "The targets to run before starting the node app.",
         "default": [],
-        "items": { "type": "string" }
+        "items": { "type": "string" },
+        "x-priority": "important"
       },
       "host": {
         "type": "string",
         "default": "localhost",
-        "description": "The host to inspect the process on."
+        "description": "The host to inspect the process on.",
+        "x-priority": "important"
       },
       "port": {
         "type": "number",
         "default": 9229,
-        "description": "The port to inspect the process on. Setting port to 0 will assign random free ports to all forked processes."
+        "description": "The port to inspect the process on. Setting port to 0 will assign random free ports to all forked processes.",
+        "x-priority": "important"
       },
       "inspect": {
         "oneOf": [
           { "type": "string", "enum": ["inspect", "inspect-brk"] },
           { "type": "boolean" }
         ],
         "description": "Ensures the app is starting with debugging.",
-        "default": "inspect"
+        "default": "inspect",
+        "x-priority": "important"
       },
       "runtimeArgs": {
         "type": "array",
         "description": "Extra args passed to the node process.",
         "default": [],
-        "items": { "type": "string" }
+        "items": { "type": "string" },
+        "x-priority": "important"
```

Comment:
I think `x-priority` shouldn't be added to most properties. You can leave the field off.